pipelines/tools-staging-prod-infra.yaml

```diff
@@ -153,17 +153,30 @@ resource_types:
     repository: govsvc/terraform-resource
     tag: 0.13.0-beta.2
 
-
+- name: github
+  type: docker-image
+  source:
+    repository: "govsvc/concourse-github-resource"
+    tag: "0.0.1551114195"
 
 
 
 resources:
 
 - name: gsp-teams
-  type: git
+  type: github
   source:
     uri: https://github.com/alphagov/gsp-teams.git
+    organization: alphagov
+    repository: gsp-teams
+    github_api_token: ""
```

Comment:
This needs to be set to something. Probably [something like this](https://github.com/alphagov/verify-eidas-pipelines/blob/958e623b8c5b59819526e357af6f9943f72d3ea9/templates/build.yaml#L41).